platform-core/platform-cache/src/main/scala/org/sunbird/cache/impl/RedisCache.scala

```diff
@@ -0,0 +1,182 @@
+package org.sunbird.cache.impl
+
+import org.apache.commons.lang3.StringUtils
+import org.slf4j.{Logger, LoggerFactory}
+import org.sunbird.cache.util.RedisConnector
+import scala.collection.JavaConverters._
+
+/**
+ * This Utility Object Provide Methods To Perform CRUD Operation With Redis
+ */
+object RedisCache extends RedisConnector {
+
+	private val logger: Logger = LoggerFactory.getLogger(RedisCache.getClass.getCanonicalName)
+
+	/**
+	 * This method store string data into cache for given Key
+	 *
+	 * @param key
+	 * @param data
+	 * @param ttl
+	 */
+	def set(key: String, data: String, ttl: Int = 0): Unit = {
+		val jedis = getConnection
+		try {
+			jedis.del(key)
+			jedis.set(key, data)
+			if (ttl > 0) jedis.expire(key, ttl)
+		} catch {
+			case e: Exception =>
+				logger.error("Exception Occurred While Saving String Data to Redis Cache for Key : " + key + "| Exception is:", e)
+				throw e
+		} finally returnConnection(jedis)
+	}
+
+	/**
+	 * This method read string data from cache for a given key
+	 * @param key
+	 * @param handler
+	 * @return String
+	 */
+	def get(key: String, handler: (String, String) => String = defaultHandler): String = {
+		val jedis = getConnection
+		try {
+			var data = jedis.get(key)
+			if (null!=handler && (null == data || data.isEmpty))
+				data = handler(key, key)
```

Comment:
If the data from the handler is not empty, we should set it to Redis.

Reply:
done. Please Review